yoojin_lee/LV_1/핸드폰-번호-가리기.js

```diff
@@ -0,0 +1,3 @@
+function solution(phone_number) {
+    return "*".repeat(phone_number.length - 4) + phone_number.slice(-4);;
+}
```

Comment:
```suggestion
function solution(phone_number) {
    return "*".repeat(phone_number.length - 4) + phone_number.slice(-4);
}
```
저랑 풀이가 같네요!!ㅎ 풀이에 세미클론(;)이 하나 더 붙어있습니다~!

Reply:
오!!!! 알려주셔서 감사합니다!!! 두개나 붙어있었는데 왜 통과가 됐을까요???🫢